integration/spark/src/main/scala/org/apache/spark/sql/CarbonEnv.scala

```diff
@@ -74,7 +75,13 @@ class CarbonEnv {
 
     sparkSession.udf.register("getTupleId", () => "")
     sparkSession.udf.register("getPositionId", () => "")
-    sparkSession.udf.register("NI", (anyRef: AnyRef) => true)
+    CreateFunctionCommand(None,
```

Comment:
please add command before this invocation to explain the modification

Reply:
done